sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -1070,15 +1070,12 @@ public async Task StartRecognizeCustomFormsWithoutLabelsCanParseMultipageForm(bo
                     expectedFirstPageNumber: expectedPageNumber, expectedLastPageNumber: expectedPageNumber);
 
                 // Basic sanity test to make sure pages are ordered correctly.
-
-                var sampleField = recognizedForm.Fields["field-1"];
                 var expectedLabelData = formIndex == 0 ? "__Address__1" : "Company Name:";
                 var expectedValueData = formIndex == 0 ? "Contoso Ltd. 2345 Dogwood Lane Birch, Kansas 98123" : "Southridge Video";
 
-                Assert.IsNotNull(sampleField.LabelData);
-                Assert.AreEqual(expectedLabelData, sampleField.LabelData.Text);
-                Assert.IsNotNull(sampleField.ValueData);
-                Assert.AreEqual(expectedValueData, sampleField.ValueData.Text);
+                FormField fieldInPage = recognizedForm.Fields.Values.Where(field => field.LabelData.Text.Contains(expectedLabelData)).FirstOrDefault();
+                Assert.IsNotNull(fieldInPage);
+                Assert.AreEqual(expectedValueData, fieldInPage.ValueData.Text);
```

Comment:
Maybe asserting that `fieldInPage.ValueData` is not null, like we do in the other test?